vendor/modules.txt

```diff
@@ -1343,6 +1343,7 @@ knative.dev/eventing/test/rekt/resources/channel
 knative.dev/eventing/test/rekt/resources/channel_impl
 knative.dev/eventing/test/rekt/resources/containersource
 knative.dev/eventing/test/rekt/resources/delivery
+knative.dev/eventing/test/rekt/resources/flaker
```

Comment:
@pierDipi this seems wrong.

especially we removed it in this repo (before) and in the upstream